one_fm/api/tasks.py

```diff
@@ -1981,6 +1997,10 @@ def initiate_checkin_notification(res):
 			""").format(recipient=recipient, shift=recipient.operations_shift, date=cstr(now_time), employee=recipient.employee, time=str(recipient.start_datetime))
 			if action_user is not None: #and not has_checkin_record(recipient.employee, recipient.log_type, res.date):
 				send_notification(title, subject, action_message, category, [action_user])
+			if action_user_id:

```

Comment:
@Talleyrand333 There is a possibility of local variable referenced before assignment error with this condition.